qa/workunits/cephtool/test.sh

```diff
@@ -197,6 +197,12 @@ function ceph_watch_wait()
     fi
 }
 
+function test_mon_osd_modify_not_exsited_option()
+{
+  ceph daemon osd.0 config set aaaaa 1 | grep "no such an option"
+  ceph daemon mon.a config set aaaaa 1 | grep "no such an option"
```

Comment:
@tchaikov 

I modified my test case by using expect_failure, but it failed:
`expect_failure $TEMP_DIR "no such an option" ceph daemon osd.0 config set aaaaa 1`
The reason is config set command will return 0 even when it is wrong.

I also tried existed case in test.sh:
```
expect_failure $TEMP_DIR "Option --osd_op_history_duration requires an argument" \
                     ceph tell osd.0 injectargs -- '--osd_op_history_duration'
```
and find it is also failed with the same reason.

Is it a known issue, or something wrong with my local env?

If it is an known issue, do you want me fix it in this pr, or just open a new one for it.